extensions/wikia/PortableInfobox/styles/PortableInfoboxMonobook_beforeSimplification.scss

```diff
@@ -0,0 +1,15 @@
+/* @TODO Remove PortableInfoboxMonobook_beforeSimplification.scss once all articles will be purged after release
+ * @see DAT-3009 purging ticket */
+// fix issues with default infobox styles on monobook
+
+.portable-infobox {
+	background-color: #f9f9f9;
```

Comment:
I know it is single use (for now) though maybe it's worth extracting to variables per scss-lint?